test/extended/olm/olmv1-webhook-support.go

```diff
@@ -0,0 +1,331 @@
+package operators
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	conditionTypeServing   = "Serving"
+	conditionTypeInstalled = "Installed"
+
+	openshiftServiceCANamespace            = "openshift-service-ca"
+	openshiftServiceCASigningKeySecretName = "signing-key"
+)
+
+var (
+	webhookTestBaseDir              = exutil.FixturePath("testdata", "olmv1", "webhook-support")
+	webhookOperatorInstallNamespace = "webhook-operator"
+)
+
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLMWebhookProviderOpenshiftServiceCA][Skipped:Disconnected][Serial] OLMv1 operator with webhooks", g.Ordered, g.Serial, func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("openshift-operator-controller-webhooks")
+
+	var cleanup func()
+
+	g.BeforeEach(func(ctx g.SpecContext) {
+		cleanup = setupWebhookOperator(ctx, oc)
```

Comment:
Could it cause flakes?
We create all resources with the same names in each test and do not wait to ensure they are fully cleaned up. Other tests use random names to avoid this.